dotcom-rendering/docs/interactives/working-with-interactives.md

```diff
@@ -0,0 +1,34 @@
+# Working with Interactives
+
+Interactive is a Design type for rich content pieces - typically with custom
+Javascript for interactive effects.
+
+DCR aims to support interactive development in a variety of ways.
+
+_Note, this document is a work in progress and more will be added shortly._
+
+## Templating
+
+Interactives support placeholders for key article metadata. This is particularly
+useful for immersive interactives, which do not come with a standard top-meta
+section, but they work for any interactive page.
+
+To use, simply add a variable to your interactive markup and DCR will populate
+it when it builds the page. A moustache-like syntax is supported:
+
+    <div>Some content with a {{ byline }} in it.</div>
+
+Unsupported variables will be left as-is (including the curlies).
+
+Supported variables are:
+
+    byline
+    sectionTag
+    standfirst
+    trailText
+    webPublicationDate
+    webTitle
+    webURL
+
+If there are others you would find useful, please contact the Dotcom team to
```

Comment:
If it is relatively easy for other developers to add to the list of variables, might we expand the documentation to include a "how to add" even if it's just a link to your PR for this?

Reply:
Yeah good point, will add. The 'ask Dotcom' point is really more about the decision-making - as anything we add will become part of a long-term contract here, so we want to be cautious and not add too much.